docs/sources/reference/components/otelcol/otelcol.receiver.otlp.md

```diff
@@ -138,12 +138,13 @@ The following arguments are supported:
 
 Name | Type | Description | Default         | Required
 ---- | ---- | ----------- |-----------------| --------
-`endpoint` | `string` | `host:port` to listen for traffic on. | `"0.0.0.0:4318"` | no
+`endpoint` | `string` | `host:port` to listen for traffic on. | `"localhost:4318"` | no
```

Comment:
I think that you need to update the integration tests config for otel to explicitly set 0.0.0.0:4318 for the receivers.

Reply:
Thank you! I was wondering why the integration test is failing and didn't think of that 😅